### DISCUSSION TITLE
GPZ (zipped GPX) files support

### DISCUSSION THREAD

**Discussion by 0709wiwiwi** [devemux86/cruiser#127] [Q&A] [answered]
**Add support for gpz (zipped gpx) files.**

- Smaller total file size saves storage space.
- Associate gpz files with your favorite GPS app.
- Transfer 100% faithfull gpx TBT gpx navigation tracks. 
- Transfer gpx sym associated icon png images plus linked full size photos.
- By the gpx sym will autodisplay the associated transfered icon png images
- By a manual trigger can show map thumbnails of the transfered photos.

Document: [Icon and image transfer by gpz](https://www.dropbox.com/scl/fi/mbukx23simpztsk814o0x/Locus-navigation-images-icons-by-gpz.doc?rlkey=2mezhsn06at03dh5q9we9krg3&dl=0)

23_01_25 Testfile gpz attached. 
[GPZ_I_xsi_bright-color-wpt_purple-navtrk_TEST_osmand_locus.gpz.txt](https://github.com/user-attachments/files/18524411/GPZ_I_xsi_bright-color-wpt_purple-navtrk_TEST_osmand_locus.gpz.txt)

**Comment by devemux86** [comment]
@0709wiwiwi thanks for the interesting idea!

We may consider it for the future depending on the other features and if it is popular with users. :slightly_smiling_face:

**Comment by 0709wiwiwi** [comment]
Update.
**File gpz.**

- **Gpx viewer 1.47** - Support for loading symbol **icons** for waypoints directly from gpz file plus attachments. **NEW**.
- **Locus map 4.26** - Support for loading symbol **icons** for waypoints directly from gpz file plus attachments. **NEW**

   _By a manual trigger both apps can also use linked photo images as map thumbnails._

![Logo_duo](https://github.com/user-attachments/assets/fa8dfdec-eeb8-4720-b3b4-91f19327f8c6)

[5r_wpt_navtrk-Cruiser-logo.gpz.txt](https://github.com/user-attachments/files/20928091/5r_wpt_navtrk-Cruiser-logo.gpz.txt)

**Reply by Ciutnik** [reply]
Thanks for the nice explanation. It's always good to learn more about something that interests us all 👍

**Comment by Logiqx** [comment]
FYI - Dan Foster (Topografix / GPX standard) is strongly in favor of a GPZ/GPX proposal that exactly mirrors the KMZ/KML behavior.

I also believe a GPZ container format (mimicking KMZ) has many pracitcal applications, so I have adapted Google's description of KMZ and tweaked it to describe GPZ. The GPZ description mimicks KMZ exactly, just referring to GPX and using GPZ examples. You can find my description of GPZ at [https://logiqx.github.io/gpx-ideas/usage/gpz/](https://logiqx.github.io/gpx-ideas/usage/gpz/).

A discussion about icons and the GPZ container format can be found in the GPX developer forum at [https://groups.io/g/gpx/message/222](https://groups.io/g/gpx/message/222). You'll need to sign up to groups.io to access the icon and GPZ discussion but if you think it's a worthwhile idea, feel free to add your thoughts to the existing message in the GPX developer forum.

Dan mentioned in another thread that he would be busy over the holiday period, but I suspect he'll be able to respond to the GPZ discussion in the near future.

Regards, Mike

**Reply by devemux86** [reply]
@Logiqx thanks for the interesting information!

I also follow the GPX forum and read all the discussions there.

**Comment by devemux86** [comment]
Import gpz (zipped gpx) files in routing and favorites will be available in the next update.

**Comment by devemux86** [comment, marked as answer]
Import gpz (zipped gpx) files is available in [Cruiser 4.0.0](https://play.google.com/store/apps/details?id=gr.talent.cruiser).

**Comment by 0709wiwiwi** [comment]
Info.

- Cruiser icons on transparent background.
[CRUISER.zip](https://github.com/user-attachments/files/25040358/CRUISER.zip)

- Cruiser gpz test file. (remove .txt)
[Cruiser_gpz-black_stroked_icons_on.transparent.gpz.txt](https://github.com/user-attachments/files/25040734/Cruiser_gpz-black_stroked_icons_on.transparent.gpz.txt)

2/3/2026.  Icons package update.

**Comment by mariush444** [comment]
What is a schedule for this good idea?
It is over 2 years old but still needed 🙂

**Reply by devemux86** [reply]
@mariush444 

Import gpz (zipped gpx) files has been available since **Cruiser 4** and now we are in **Cruiser 5**. :slightly_smiling_face: 

The announcement is above as answer to this thread:
- https://github.com/devemux86/cruiser/discussions/127#discussioncomment-9312170

**Reply by devemux86** [reply]
The original thread idea is for import gpz (zipped gpx) files.
This was implemented a few years ago.

Please avoid constantly editing thread descriptions, as this confuses users.
For feature requests, it is best to create new threads.

We can then discuss the proposal and depending on how many users want it, we can consider it.

**Reply by mariush444** [reply]
I'm sorry, this is my mistake. 
You absolutely right
